code/game/machinery/flasher.dm

```diff
@@ -124,7 +124,10 @@
 			continue
 
 		if(L.flash_act(affect_silicon = 1))
```

Comment:
what does this do that we need a second flash_borg too?

Reply:
flash_act only interacts with the eyes- it blurs/blinds/applies the flash overlay and also checks for flash protection, returning whether it was successful or not